### PR TITLE
Add release notes for Mender 3.4.0 release

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: 20.Mender-client/docs.md [hunk 1]
```diff
@@ -8,6 +8,105 @@ github: false
...
+* If paused before ArtifactReboot, and then manually
+  rebooting the device outside of Mender, the client will properly
+  resume the update now, instead of failing and rolling back.
+  ([MEN-5709](https://tracker.mender.io/browse/MEN-5709))
+* Append log entries to syslog at the correct level
+* The client update and inventory checks are now unaffected by the use
+  of the `check-update` and `send-inventory` commands. While previously, this
+  would both move the intervals at which checks we're done, and also extend them
+  beyond the expected polling intervals configured.
+  ([INF-1000](https://tracker.mender.io/browse/INF-1000), [MEN-5547](https://tracker.mender.io/browse/MEN-5547))
```
Comment (oleorhagen) [inline 20.Mender-client/docs.md:97]: Q: What's the `INF-1000` doing here? :thinking:

File: 20.Mender-client/docs.md [hunk 2]
```diff
@@ -8,6 +8,105 @@ github: false
...
+* Resolve symlinks for /dev/disk/by-partlabel
+
+##### Features
+
+* Add DaemonLogLevel parameter in configuration file
+  ([MEN-5583](https://tracker.mender.io/browse/MEN-5583))
+
+##### Other
+
+* Upgrade mender-artifact library to latest
```
Comment (oleorhagen) [inline 20.Mender-client/docs.md:107]: God, horrible commit message (I think it might be me even), should we fix?

File: 21.mender-connect/docs.md [hunk 3]
```diff
@@ -8,6 +8,54 @@ github: false
...
+
+### Changelogs
+
+#### mender-connect (2.1.0)
+
+New changes in mender-connect since 2.0.2:
+
+##### Features
+
+* if /etc/shells file does not exist, assume /bin/sh is listed
```
Comment (oleorhagen) [inline 21.mender-connect/docs.md:56]: Odd feature

File: 30.mender-artifact/docs.md [hunk 4]
```diff
@@ -8,6 +8,76 @@ github: false
...
+* Implement write for empty payload artifacts
+  ([MEN-2586](https://tracker.mender.io/browse/MEN-2586))
+* Add possibility to read bootstrap artifacts
+  ([MEN-2586](https://tracker.mender.io/browse/MEN-2586))
+* Add support for provides and depends flags in write and read
+  ([MEN-2586](https://tracker.mender.io/browse/MEN-2586))
+* Add PKCS#11 standard support for artifacts signing
+  ([MEN-5759](https://tracker.mender.io/browse/MEN-5759))
+
+##### Dependabot bumps
```
Comment (oleorhagen) [inline 30.mender-artifact/docs.md:66]: Unrelated to this of course, but should we change this to something like: `depenency updates`?

File: 32.mender-convert/docs.md [hunk 5]
```diff
@@ -8,6 +8,55 @@ github: false
...
+### Changelogs
+
+#### mender-convert (3.0.1)
+
+New changes in mender-convert since 3.0.0:
+
+##### Bug Fixes
+
+* Detect when EFI is not available and disable grub.d integration.
+* number of parameters to accept the optional parameters
```
Comment (oleorhagen) [inline 32.mender-convert/docs.md:57]: Something missing here? :thinking:

File: 50.mender-binary-delta/docs.md [hunk 6]
```diff
@@ -7,6 +7,23 @@ shortcode-core:
...
+
+### Changelogs
+
+#### mender-binary-delta (1.4.1)
+
+New changes in mender-binary-delta since 1.4.0:
+
+##### Features
+
+* Licenses are now available in the package, instead of only
```
Comment (oleorhagen) [inline 50.mender-binary-delta/docs.md:23]: Hmm, why the bump?

File: 52.mender-gateway/docs.md [hunk 7]
```diff
@@ -8,6 +8,22 @@ github: false
...
+
+### Changelogs
+
+#### mender-gateway (1.0.1)
+
+New changes in mender-gateway since 1.0.0:
+
+##### Features
+
+* Licenses are now available in the package, instead of only
```
Comment (oleorhagen) [inline 52.mender-gateway/docs.md:23]: Related to above: Here we list it as a feature (also), but we do a patch bump?

**Review by oleorhagen — APPROVED** [review]
Approving, but left a few questions and suggestsions :)

File: 20.Mender-client/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 20.Mender-client/docs.md:97]: It comes from [here](https://github.com/mendersoftware/mender/commit/f27561fdf2c9e9d5b2f1d526d08615e1ec07865c). I'll clean it up.

File: 20.Mender-client/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 20.Mender-client/docs.md:107]: It is mine, but I don't see it that horrible. Do you want something like `Upgrade mender-artifact library to v0.0.0-20220913084855-9ed8ad0d53d0`?

File: 21.mender-connect/docs.md (hunk 3 above)
Comment (lluiscampos) [reply 21.mender-connect/docs.md:56]: Contributed feature, PR [here](https://github.com/mendersoftware/mender-connect/pull/69). Should we rename? Do you have a suggestion?

File: 20.Mender-client/docs.md (hunk 2 above)
Comment (oleorhagen) [reply 20.Mender-client/docs.md:107]: Yeh, that was my thinking

File: 30.mender-artifact/docs.md (hunk 4 above)
Comment (lluiscampos) [reply 30.mender-artifact/docs.md:66]: I like that suggestion - `bumps` is very informal.

File: 32.mender-convert/docs.md (hunk 5 above)
Comment (lluiscampos) [reply 32.mender-convert/docs.md:57]: Coming from [here](https://github.com/mendersoftware/mender-convert/commit/102ba1fb3000830b7580542a221f521d1f4f85f3). Renaming to `Amend error message with optional parameters`

File: 21.mender-connect/docs.md (hunk 3 above)
Comment (oleorhagen) [reply 21.mender-connect/docs.md:56]:
```
if the `/etc/shells` file does not exit, fall back to the default shell
```

File: 50.mender-binary-delta/docs.md (hunk 6 above)
Comment (lluiscampos) [reply 50.mender-binary-delta/docs.md:23]: In my opinion this is not a feature, but it was marked as such in the commit. I will manually downgrade it to `Other`

File: 52.mender-gateway/docs.md (hunk 7 above)
Comment (lluiscampos) [reply 52.mender-gateway/docs.md:23]: Same as above

File: 30.mender-artifact/docs.md (hunk 4 above)
Comment (oleorhagen) [reply 30.mender-artifact/docs.md:66]: https://github.com/mendersoftware/integration/pull/2174

Comment (lluiscampos) [reply 30.mender-artifact/docs.md:66]: Thanks!

**Pull request merged**